.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 /api/dist
 /api/coverage
 .env
+.vscode
```

Comment:
Dunno if this never left, (because the thread is marked as resolved) but it's still here

Reply:
@aagm can you remove this before we merge?